spec/factories/campaigns.rb

```diff
@@ -19,10 +19,19 @@
 FactoryGirl.define do
   factory(:campaign) do
     volunteer_center
-    name 'MyString'
-    description 'MyString'
-    donation_dateline Date.today + 7
+    name 'Share Your Christmas 2015'
+    description 'Share Your Christmas Campaign 2015'
+    donation_deadline Date.today + 7
     reminder_date Date.today + 14
-    logo { fixture_file_upload(Rails.root.join('public', 'images', 'original', 'missing.png')) }
+    # logo { fixture_file_upload(Rails.root.join('public', 'images', 'original', 'missing.png')) }
   end
+
+  factory(:campaign_thanksgiving, class: Campaign) do
```

Comment:
You can also do this as a trait rather than a separate factory. A trait has the benefit of being clear about code re-use (it only overrides the attributes inside the trait)

```
# this gets nested inside the campaign factory
trait :thanksgiving do
  name 'Thanksgiving 2015'
  description 'Thanksgiving Campaing 2015'
  donation_deadline Date.today + 107
  reminder_date Date.today + 114
end
```

You might call it like this: `let(:turkeyday) { FactoryGirl.create :campaign, :thanksgiving }`